### ISSUE TITLE
Question: Possible bug

### ISSUE THREAD

**Issue by jdsika** [TalaoDAO/AltMe#2169]
Hi team,

first of all: keep up the great work!

In the interaction with you a question came up which might be a bug in altme.

## Example credential:
https://github.com/ASCS-eV/credentials/blob/main/examples/member-credential-full.json

## Specification:
https://www.w3.org/TR/vc-data-model/#issuer

> The value of the issuer [property](https://www.w3.org/TR/vc-data-model/#dfn-property) MUST be either a [URI](https://www.w3.org/TR/vc-data-model/#dfn-uri) or an object containing an id [property](https://www.w3.org/TR/vc-data-model/#dfn-property). It is RECOMMENDED that the [URI](https://www.w3.org/TR/vc-data-model/#dfn-uri) in the issuer or its id be one which, if dereferenced, results in a document containing machine-readable information about the [issuer](https://www.w3.org/TR/vc-data-model/#dfn-issuers) that can be used to [verify](https://www.w3.org/TR/vc-data-model/#dfn-verify) the information expressed in the [credential](https://www.w3.org/TR/vc-data-model/#dfn-credential).

## My custom issuer type in the context:
```
"AscsIssuer": {
                "@id": "https://schema.ascs.digital/AscsMemberCredential/v1#AscsIssuer",
                "@context": {
                    "@version": 1.1,
                    "@protected": true,
                    "name": "https://schema.org/name",
                    "url": "https://schema.org/url"
                }
            },
```

## My issuer in the credential:
```
"issuer": {
        "id": "did:pkh:tz:tz1ggujjYjA7oYoaZBzTg1tYSXn3VMjcgDuv",
        "type": "AscsIssuer",
        "name": "Automotive Solution Center for Simulation e.V.",
        "url": "https://identity.ascs.digital/"
    },
```

## Assumption:
If I try the credential workflow with altme the application is crashing and/or not successfully exchanging credentials. I think that my credential is according to specification and therefore I am asking if that may be an issue in the altme application?

Best regards
Carlo

**Comment by ThierryThevenet** [comment]
Yes you are right, there is a bug when $.issuer is a json object and not a string (uri).
The issue is about the credential display, i have seen that when when we used the credential manifest with the impossibility to display the issuer name through the path $.issuer.name

**Comment by ThierryThevenet** [comment]
Test :  https://talao.co/sandbox/direct_offer?VC=AscsMemberCredential.json

**Comment by jdsika** [comment]
Yes, of course. It just took me a little time to figure out that you modified the proposed credential at that specific point because of the big :))

**Comment by ThierryThevenet** [comment]
Yes it is correct, the second form with issuer as a json object is correct but it is not supported by Altme.

If you need it let me know ?
@jdsika

**Comment by hawkbee1** [comment]
@jdsika Do you need this functionality?

**Comment by jdsika** [comment]
I can move around it! I do not know if this is still a topic if we use jwt-vc ?
If that type of structure is still valid in the jwt-vc format I would consider this on the feature list but with a low priority.

**Comment by ThierryThevenet** [comment]
It does not really matter in the jwt format as this "issuer" attributes is replaced by the "iss" attribute of the jwt.

**Comment by jdsika** [comment]
And there I can create a nested structure?

**Comment by ThierryThevenet** [comment]
"iss" is a string
Best will be to add in the vc json object  of the jwt an "issuedBy" attribute with all the information .

**Comment by jdsika** [comment]
Then I would see it as a low priority feature because the only impact is a potential user being confused by the error when trying to use the object.
If you plan to not support then the ultimate and maybe not necessary step is to clarify in your documentation is what subset of the (many different) standards and documents you support. But... Maybe we are optimizing here :)

**Comment by ThierryThevenet** [comment]
Ok, lets do that the ldp_vc format  (pure json-ld with ld proof) is less and less used in EU.

**Comment by ThierryThevenet** [comment]
@bibash28 Test on OIDC4VCI Test 5, choose EmailPass2 with profile DIIP V3.0


**Be careful with regression issues -> retest all issuance with ldp_vc format**

**Comment by ThierryThevenet** [comment]
see below VC to store and display

```
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    {
      "EmailPass2": {
        "@context": {
          "@protected": true,
          "@version": 1.1,
          "email": "schema:email",
          "id": "@id",
          "schema": "https://schema.org/",
          "type": "@type"
        },
        "@id": "https://github.com/TalaoDAO/context#emailpass2"
      },
      "name": "https://schema.org/name"
    }
  ],
  "id": "urn:uuid:ef5af892-b054-4990-b9a2-208970c951ed",
  "type": [
    "VerifiableCredential",
    "EmailPass2"
  ],
  "credentialSubject": {
    "id": "did:key:zQ3shuCH4atYWSjWvBhat5hkS98p4QzBeeXJtBRf22aAbW4vP",
    "type": "EmailPass2",
    "email": "john.doe@gmail.com"
  },
  "issuer": {
    "id": "did:key:zQ3shPyHAZTwTTD4JFHvm7Q2tTKUeAep8mrt2Rp48wK4A2YME",
    "name": "tatao"
  },
  "issuanceDate": "2024-11-04T18:11:16Z",
  "proof": {
    "type": "EcdsaSecp256k1Signature2019",
    "proofPurpose": "assertionMethod",
    "verificationMethod": "did:key:zQ3shPyHAZTwTTD4JFHvm7Q2tTKUeAep8mrt2Rp48wK4A2YME#zQ3shPyHAZTwTTD4JFHvm7Q2tTKUeAep8mrt2Rp48wK4A2YME",
    "created": "2024-11-04T17:11:16.892Z",
    "jws": "eyJhbGciOiJFUzI1NksiLCJjcml0IjpbImI2NCJdLCJiNjQiOmZhbHNlfQ..0ve9NhnmFzRqLu1UCyeALeveGAjVyM0Rdf_G0wIPm2w6EXMb1PmkNqARg-fK2xV_pbhEcCDM-_ZSk8LjQK_Wkw"
  },
  "expirationDate": "2024-11-16T18:11:16Z",
  "issued":
``` "2024-11-04T18:11:16Z"
}
